packages/apollos-ui-kit/src/inputs/Radio/Radio.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import RadioButton from './RadioButton';
+
+export default class Radio extends Component {
```

Comment:
I know you were just moving the files around but generally we like to put all exports at the bottom of the file so we always know where to find them. 

```suggestion
class Radio extends Component {
```

Reply:
you could do a similar change to the other components in the `/Radio` directory too if you wanted.